README.md

```diff
@@ -8,12 +8,289 @@ The executables found on the [release tab of this repository](https://github.com
 
 The script `interoperability_report.py` generates automatically
 the verification between two executables of these interoperability tests.
-The tests that the script runs must be defined previosly (for example
+The tests that the script runs must be defined previously (for example
```

Comment:
could i describe the dictionary format in test_suite.py and then here put: 'The tests that the script runs must be defined previously in a python file following the pattern described in `test_suite.py`. By default it will run the tests from `test_suite.py`.'.
In this way there is not the same information in two places, and the readme is not that long

Reply:
done